src/DataTypes/DataTypeAggregateFunction.cpp

```diff
@@ -159,7 +159,7 @@ SerializationPtr DataTypeAggregateFunction::doGetDefaultSerialization() const
 }
 
 
-static DataTypePtr create(const ASTPtr & arguments)
+static DataTypePtr create(const ASTPtr & arguments/* proton: starts */, bool compatible_with_clickhouse = false/* proton: ends */)
```

Comment:
For this one liner, we can just mark it something like this

```
static DataTypePtr create(const ASTPtr & arguments, bool compatible_with_clickhouse = false) /// proton : updated
```

Reply:
Good to know!